### PR TITLE
fix(funman): output should be a temp config and save for reuse makes it a non temp config

### PR DESCRIPTION
**Pull request by shawnyama**
# Description
- fix save for reuse
- temporary config in funman drilldown must be saved for reuse to be accessed in the config operator

https://github.com/user-attachments/assets/b1d8a142-6d7d-41d9-9770-c5e3c73cf251

### CODE REVIEW

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ValidateModelConfigHandler.java [hunk 1]
```diff
@@ -128,12 +128,19 @@ public TaskResponse onSuccess(final TaskResponse resp) {
 				props.newModelConfigName,
 				contractedModel.getDescription()
 			);
+			contractedModelConfiguration.setTemporary(true); // Config should only be accessible within the operator for now
```
Comment (mwdchang) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ValidateModelConfigHandler.java:131]: I would reword or just remove this comment, as the notions of workflow operators are decouples from that of the taskrunner, you could in theory use the task runners without any workflows and the comment will make no sense in that context.

File: packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue [hunk 2]
```diff
@@ -819,10 +821,12 @@ async function renderCharts() {
 	updateStateCharts();
 	updateParameterCharts();
 
+	if (!validatedModelConfiguration.value) return;
+
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue:825]: This function feels a bit odd and maybe some reorganization would be good? Are you actually wanting to return or do we want to throw an error? 

But moreover, the underlying model and MMT do not change across validation runs, so `getModel`/`getMMT` are likely better if they are in onMounted hook since they need to be loaded once and once-only. What is the reason we only get results and in "renderCharts"?

File: packages/client/hmi-client/src/services/models/funman-service.ts [hunk 3]
```diff
@@ -122,8 +122,13 @@ export const processFunman = (result: any, onlyShowLatestResults: boolean) => {
 	const boxes: FunmanBox[] = [];
 	const trajectories: any[] = [];
 
+	console.log('trueBoxes', trueBoxes);
+	console.log('falseBoxes', falseBoxes);
+	console.log('ambiguousBoxes', ambiguousBoxes);
```
Comment (jryu01) [inline packages/client/hmi-client/src/services/models/funman-service.ts:127]: We should not leave console.log on the committed code.

File: packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue (hunk 2 above)
Comment (shawnyama) [reply packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue]: I did the clean ups and I figured out a way to just process the funman results once. I was filtering the data depending on the "Show furthest results checkbox", but I decided to just mark which boxes are the furthest to use them later.

**Review by dvince2 — APPROVED** [review]

**Review by mwdchang — APPROVED** [review]
thanks!
